scipy/stats/contingency.py

```diff
@@ -233,13 +233,59 @@ def chi2_contingency(observed, correction=True, lambda_=None):
     .. [3] Cressie, N. and Read, T. R. C., "Multinomial Goodness-of-Fit
            Tests", J. Royal Stat. Soc. Series B, Vol. 46, No. 3 (1984),
            pp. 440-464.
+    .. [4] Berger, Jeffrey S. et al. "Aspirin for the Primary Prevention of
+           Cardiovascular Events in Women and Men: A Sex-Specific
+           Meta-analysis of Randomized Controlled Trials."
+           JAMA, 295(3):306-313, :doi:`10.1001/jama.295.3.306`, 2006.
 
     Examples
     --------
-    A two-way example (2 x 3):
+    In [4]_, the use of aspirin to prevent cardiovascular events in women
+    and men was investigated. The study notably concluded:
+
+        For women and men, aspirin therapy reduced the risk of a composite of
+        cardiovascular events due to its effect on reducing the risk of
+        ischemic stroke in women [...]
+
+    The article list multiple studies for various cardiovascular events. Let's
+    focus on the ischemic stoke in women.
+
+    The following table summarizes the results of the experiment in which
+    took aspirin or a placebo on a regular basis for several
+    years. Occurences of ischemic stroke were recorded::
+
+                          Aspirin   Control/Placebo
+        Ischemic stroke     176           230
+        No stroke         21035         21018
+
+    Is there evidence that the aspirin reduces the risk of ischemic stroke?
+    We can formulate a null hypothesis :math:`H_0`:
+    "the effect of aspirin is equivalent to that of placebo".
+    Let's assess the plausibility of this hypothesis with
+    a chi-square test.
 
     >>> import numpy as np
     >>> from scipy.stats import chi2_contingency
+    >>> table = np.array([[176, 230], [21035, 21018]])
+    >>> res = chi2_contingency(table)
+    >>> res.statistic
+    6.892569132546561
+    >>> res.pvalue
+    0.008655478161175739
+
+    Using a significance level of 5%, we would reject the null hypothesis in
+    favor of the alternative hypothesis: "the effect of aspirin
+    is not equivalent to the effect of placebo".
+    Because `stats.contingency.chi2_contingency` performs a two-sided test
+    (i.e. the alternative hypothesis does not indicate the direction of the
+    effect), we can use `stats.contingency.odds_ratio` to support the
+    conclusion that aspirin *reduces* the risk of ischemic stroke.
```

Comment:
```suggestion
    Because `scipy.stats.contingency.chi2_contingency` performs a two-sided
    test, the alternative hypothesis does not indicate the direction of the
    effect. We can use `stats.contingency.odds_ratio` to support the
    conclusion that aspirin *reduces* the risk of ischemic stroke.
```